nsis/data/languages/Nsis.sla

```diff
@@ -5,7 +5,7 @@
 <space name="ram" index="3" bigendian="false" delay="1" size="8" physical="true"/>
 <space name="register" index="4" bigendian="false" delay="0" size="4" physical="true"/>
 </spaces>
-<symbol_table scopesize="68" symbolsize="416">
+<symbol_table scopesize="72" symbolsize="440">
```

Comment:
Double checking, Nsis.sla is auto generated right? So changes here are likely fine?

Reply:
Yes all those changes were generated